### DISCUSSION TITLE
OGO and GWG Logo proposal

### DISCUSSION THREAD

**Discussion by iboonox** [open-gitops/project#88] [General]
From [issue](https://github.com/gitops-working-group/gitops-working-group/issues/61#issuecomment-787476282)

<img width="673" alt="1" src="https://user-images.githubusercontent.com/1584813/109497620-9e06ed00-7a92-11eb-80ed-dc128260b992.png">

![2](https://user-images.githubusercontent.com/1584813/109498009-2ab1ab00-7a93-11eb-8ef8-f42b649879c2.png)

![sss](https://user-images.githubusercontent.com/1584813/109500375-7c0f6980-7a96-11eb-8f73-5708ec6839bc.png)

**Comment by schlomo** [comment]
I like the look of this. Both color schemes agree with me. It is good to mention the Working Group in the logo to signal that this is the logo for the working group specifically and not GitOps in general.

The only thing I would consider changing is also providing a square version, e.g. by putting the text under the image. Manye usage scenarios require a square version.

**Reply by scottrigby** [reply]
@iboonox re formats, see https://github.com/cncf/artwork for the required formats:
> We prepare artwork in 2 formats (PNG & SVG), 3 layouts -- horizontal (also known as landscape format), stacked (which is closer to square), and icon (which does not include the name and is square), and at least 3 versions (color/black/white). So, that's at least 18 versions of most logos.

**Reply by iboonox** [reply]
Once the proposal is validated, we could generate the other formats yes.

**Reply by scottrigby** [reply]
Status: there has only been enthusiastic and constructive/collaborative feedback so far. And everyone seems happy with the most recent iteration: https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-478223

We have already used this as a provisional design for the GitHub org, and Twitter account. See https://github.com/gitops-working-group/gitops-working-group/issues/96.

@iboonox What is your schedule to work on a PR for https://github.com/cncf/artwork? See https://github.com/cncf/toc/issues/584#issuecomment-806190116 for priority. If you are able to, that is great. But if you will be too busy to create the formats for the PR in time, in case that would you mind passing the design source files to someone who would be available?

**Reply by TracyRagan** [reply]
Looks great - we need to make sure it still looks great in an all white or all black versions.  If so, thumbs up from me.

**Comment by tonit** [comment]
Wasn't there a discussion about getting rid of the "working group" part once being a CNCF Sandbox project (thats a near term goal right?)  ? On slack I liked the  new name "GitOps Initiative" proposal.
+1 for a square version, too.

**Reply by scottrigby** [reply]
Yes I suggested this in a WG meeting ([see meeting agenda](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.w91ueqcmi7nz)), and everyone on the call gave enthusiastic support. However I will next follow my own advice and create a GH Discussion proposal for that, and loop in the necessary people. Thanks for the reminder @tonit 💯

**Reply by tonit** [reply]
Loop me in if you can. Don't know how the rules are over here but I'd like to help were possible.

**Reply by scottrigby** [reply]
Started discussion https://github.com/gitops-working-group/gitops-working-group/discussions/64

**Reply by scottrigby** [reply]
The name change proposal is complete 🎉  https://github.com/gitops-working-group/gitops-working-group/discussions/64#discussioncomment-505062

We will ultimately want variants for:
- GitOps Working Group
- OpenGitOps

@iboonox or anyone else, do you have thoughts on whether these should have different color schemes? Or should they just have different text but otherwise the same? I'm also thinking when it's logo-only, such as a GitHub org that shows on a user's profile. Should those look identical? I'm not personally opposed either way

**Comment by bia** [comment]
I really like the logo @iboonox !

@scottrigby also encouraged me to share these, which I made some months back. If they inspire you, we can combine them in some way. :)

![FINAL201006gitops1](https://user-images.githubusercontent.com/92439/110022102-d117dd80-7d2b-11eb-9a26-5760720f6867.png)
![gitops-logo-transparent@2x](https://user-images.githubusercontent.com/92439/110022151-dbd27280-7d2b-11eb-9420-ac50479d3499.png)

**Reply by scottrigby** [reply]
Thanks @bia! It's great to see other ideas in the mix too 😄

**Reply by iboonox** [reply]
Nice work @bia.
there is a lot of detail in each version, i don't really see how we can merge them but i will try

**Comment by christianh814** [comment]
+1 on the @iboonox logo! Looks great!

**Comment by scottrigby** [comment]
@iboonox Echoing everyone else, I quite like this proposed logo 👏

A few additional considerations/suggestions:

1. As @tonit mentioned in https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-418201 we may want to do a version of this without the "working group" label as well
1. What font is this? If the font isn't published under an open source license, could you find another one that is? Or could someone else recommend one? As @bia mentioned in https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-430694 iterating on this can be a collaborative effort while still acknowledging the work each person has contributed
1. I really like your initial inspiration for the simple line abstraction referencing data/configuration (not dissimilar to how [GitHub Codespaces](https://github.com/features/codespaces) does). IMO the unbroken diagonal lines in the current logo iteration look sleek, but I feel the horizontal lines with spaces in your initial idea convey an image of configuration/data/code much more clearly. WDYT?

<img width="102" alt="Screen Shot 2021-03-04 at 10 13 32 PM" src="https://user-images.githubusercontent.com/407675/110062228-df1f2b80-7d36-11eb-9ac1-fb329f563f30.png"><img width="93" alt="Screen Shot 2021-03-04 at 10 07 33 PM" src="https://user-images.githubusercontent.com/407675/110061700-075a5a80-7d36-11eb-9d55-d46a126d7447.png">

**Reply by bia** [reply]
Here is a _very_ rough sketch that misses out on all of the cool ways that @iboonox had aligned the two circles :
![210305gitops-logo](https://user-images.githubusercontent.com/92439/110181009-2a0a7300-7e0b-11eb-96a2-ab7c7bf65cc4.png)

**Reply by iboonox** [reply]
@scottrigby I agree with the horizontal lines,  this is what i tried to do in the first iteration
[slack post](https://cloud-native.slack.com/files/UEB4FKKNF/F01NU0GH90A/workflow1.png )

**Reply by iboonox** [reply]
What about this, it needs improvements ...

![gitopsinit](https://user-images.githubusercontent.com/1584813/110312793-dca62580-8005-11eb-8737-33c398f91fe4.png)

**Reply by tonit** [reply]
Thanks @iboonox for the iterations! Good progress. I do like the graphical parts. Nice!
I'd challenge the font for "Initiative" though:
- Is there a reason why this font? 
- Also I think i'd be nice to see a version where "Initiative" without separate background

WDYT?

**Reply by schlomo** [reply]
Might be worth to also try an all-caps version of INITIATIVE.

**Reply by schlomo** [reply]
I like your original suggestion and also the new one with the horizontal lines and more peaceful colors, @iboonox

**Reply by iboonox** [reply]
> Thanks @iboonox for the iterations! Good progress. I do like the graphical parts. Nice!
> I'd challenge the font for "Initiative" though:
> 
> * Is there a reason why this font?
> * Also I think i'd be nice to see a version where "Initiative" without separate background
> 
> WDYT?
<img width="64" alt="ppppppppppppp" src="https://user-images.githubusercontent.com/1584813/110611389-76e5a500-818f-11eb-8d9c-3a57a05546fa.PNG">


@tonit Not particularly, I liked the shape of the letter G which refers to an arrow somehow (  like the arrows of the logo )

**Reply by iboonox** [reply]
<img width="1059" alt="text" src="https://user-images.githubusercontent.com/1584813/110611496-8fee5600-818f-11eb-8993-99380e7927bf.PNG">

I 'd like to have your opinion on the text placement, we can change the font later if necessary

**Reply by scottrigby** [reply]
Love the tag-teaming! 

I think this does read more like configuration/code now 👌 

I'm not opinionated about font, as long as it's open license and readable at various sizes.

My only feedback on text placement is similar to the above. Probably better to have it span the full width below "GitOps" for the sake of readability when small.

As part of the name proposal (still in progress), I would love to see one for both GitOps "Working Group" and GitOps "Initiative". I also think the color background on both of those looks label-like, which feels very fitting. Just a thought: erhaps each of those two can have a different label color? The text is good even for color blindness, and different colors between the two could be good to otherwise see the difference even at a quick glance. Just a thought.

**Reply by scottrigby** [reply]
Also IMO definitely capital G and O, not all caps. I don't have an opinion on making "Git" and "Ops" color-separated or all one color. Personally one color feels nice to me, as it helps send the message this is the way you spell "GitOps" - all one word 😺

**Reply by christianh814** [reply]
I absolutely LOVE this @iboonox
https://user-images.githubusercontent.com/1584813/110312793-dca62580-8005-11eb-8737-33c398f91fe4.png

**Comment by tonit** [comment]
This:
![image](https://user-images.githubusercontent.com/67375/110647616-423a1380-81b8-11eb-9e2e-0c943ff5ac56.png)
seems most attractive to me but probably does not work as a logo in other contexts.
Still, i really like this the most from the set.

This comes second:
![image](https://user-images.githubusercontent.com/67375/110647903-8f1dea00-81b8-11eb-9967-29d68760ac3a.png)
but its less iconic. Almost just the (not iconic by itself) font. 

In any case, all of those:
![image](https://user-images.githubusercontent.com/67375/110648075-b07ed600-81b8-11eb-9258-43d115bc447c.png)
are superior to the other options for me.

About all caps for "initiative" or not I am not really sure. The not-all-caps version looks slightly better to me though.

Maybe we can talk/discuss about a vote in tomorrows call? Or how will this be decided?

**Reply by tonit** [reply]
What happened to this? 
![image](https://user-images.githubusercontent.com/67375/110648644-2daa4b00-81b9-11eb-8987-01d014e80dec.png)

That graphic combined with the text-style from the last submissions would be nice.

**Reply by scottrigby** [reply]
Cross-linking https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-463375 about CNCF artwork format combinations

**Reply by iboonox** [reply]
![image](https://user-images.githubusercontent.com/1584813/111024838-15b51000-83e1-11eb-97a3-bae286682400.png)
@tonit it's not the logo it's just the text

**Reply by iboonox** [reply]
I tried to take your comments into account, the text "community" is just an example we can change it later
<img width="929" alt="uuuu" src="https://user-images.githubusercontent.com/1584813/111035330-3ea5c700-841a-11eb-8d9b-9561689ae0d0.PNG">


@bia your feedback is always appreciated. :)

**Reply by roberthstrand** [reply]
I think this looks really, really good 👍🏻

**Reply by tonit** [reply]
Wow really nice!

**Comment by bricef** [comment]
@iboonox Just wanted to say again how cool the logo is. Don't have specific feedback, but I really think you captured the concept of GitOps in the logo ❤️

**Comment by iboonox** [comment]
As suggested by @scottrigby , here are the logos proposals for the two projects

OGO : Open GitOps
GWG : GitOps Working Group

If you have any comment or feedback please do not hesitate.

<img width="945" alt="10" src="https://user-images.githubusercontent.com/1584813/112864020-bb41d200-90b7-11eb-9efe-db6989c1c472.png">

<img width="945" alt="20" src="https://user-images.githubusercontent.com/1584813/112864043-bed55900-90b7-11eb-8f94-2f7783ebcdb0.PNG">

<img width="936" alt="30" src="https://user-images.githubusercontent.com/1584813/112864053-c1d04980-90b7-11eb-89e8-14b8cc312231.PNG">

<img width="945" alt="60" src="https://user-images.githubusercontent.com/1584813/112868628-90a64800-90bc-11eb-8aa0-9d86cdb24e90.PNG">

**Reply by scottrigby** [reply]
Thanks @iboonox ✨ 

We are still timboxing consensus for logo/artwork by 5PM PDT this Wednesday, March 31.

Last Thursday I posted the email subject `[gitops-wg] Artwork consensus` to the cncf-sig-app-delivery@lists.cncf.io mailing list, with this content:

>Hello again 👋
>
>Please voice your thoughts – concerns, praise or otherwise – on the artwork design so far:
>- Discussion open-gitops/gitops-working-group#63 (active for 24 days. Note: click "read more" on long discussion threads)
>- Issue #100 (includes latest iterations)
>
>This is important for CNCF project onboarding. In the last GitOps WG meeting we agreed to timebox the logo/artwork consensus. Let's timebox responses **until 5PM PT on March 31**.
>
>Either comments or emoji reactions are very welcome!
>- 👀 means you're looking (but we won't necessarily wait for another response)
>- 👍😀🎉❤️🚀 means yes, with varying levels of enthusiasm (feel free to use more than one!) 
>- 👎😕 means you have a legitimate concern – please specify what/why in a discussion comment for the issue to be discussed and considered
>
>Thanks for participating ✨🗳


There has only been enthusiastic and constructive/collaborative feedback so far, which has been worked into the design. Everyone seems happy with the previous iteration for the `GitOps Working Group` logo: https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-478223 (we've already used this as a provisional design for the GitHub org, and Twitter accounts. See #96). This most recent iteration by @iboonox is a refinement of all of that, and introduces the `OpenGitOps` companion logo, side by side.

**Reply by bricef** [reply]
I've got a soft spot for the two arrow logo, but the OGO/OpenGitOps logo looks great too. 

Immediately recognisable, Clearly connected to the GitOps WG logo, subtle but distinct colour differences. Typography for text is clear.

Two things I might do:
1. Make the G look more like a circling arrow in the OGO logo
2. Reduce the number of colours overall by reusing an exiting colour for the dark text instead of introducing a new one.

Those are very minor things though and I'd be happy to see both logos adopted without modification ❤️

**Reply by bricef** [reply]
PS: When can we get T-shirts? 😄

**Reply by tonit** [reply]
I do like the logos, particularly the (to me new) OpenGitOps Versions. I think they are really clever: "OGO" for short, the arrow (reconciliation loop) within the  "G". classy!

One note, the "G" in OGO reminds me about the "G" logo Google used (or still uses?). Is this the same font? 
Maybe there is a chance to make it look less Googly? Its not a problematic thing to me, just an observation.

Great work, @iboonox !!

**Reply by scottrigby** [reply]
Once the window for feedback is up (tomorrow), perhaps we should wait another week (Weds April 7th) before submitting the artwork PR itself. This would allow for more subtle tweaks if necessary. The reason for this suggestion is – project logos _could_ be updated later, but CNCF prefers not to update these frequently, or too many times - mainly to promote project brand stability.

Here are the areas to consider before finalizing:
1. The shape
    - Are we satisfied with the GWG and OGO logo outlines themselves?
    - It appears everyone likes the concept, and it's looking great!
    - One note is @bricef 's comment above:
        >Make the G look more like a circling arrow in the OGO logo
2. Color
    - Are we satisfied with the color of both GWG and OGO logos? 
    - @iboonox maybe we can look at GWG and OGO color swatches side-by-side? and decide on this separately?
3. Font for the text?
    - One note was @tonit's comment:
        >Maybe there is a chance to make it look less Googly?
    - IMO the most important thing is to ensure whichever font people like the most is an open source font

@iboonox thanks for all your work on this! ❤️

**Reply by scottrigby** [reply]
PS To see how the current iteration looks, I've updated the provisional logo on https://twitter.com/OpenGitOps, with a matching banner image that @iboonox whipped up, and on https://github.com/open-gitops (you may need to clear your browser image cache)

**Reply by iboonox** [reply]
@bricef 
> Make the G look more like a circling arrow in the OGO logo  

It's a bit complicated to keep the G readable with two arrows, if you have an example ( just a quick sketch ) i'm interrested! :)

@tonit 
> One note, the "G" in OGO reminds me about the "G" logo Google used (or still uses?). Is this the same font?

No, it not a font, just a shape but i agree the "G"  looks like the google "G". We will try another font 👍

**Reply by iboonox** [reply]
Thank you @scottrigby for your help and for the modification  🥇 
It was a little tricky  😄  but you made it 👍 

![image](https://user-images.githubusercontent.com/1584813/113044753-51502800-919e-11eb-8da2-41a18ae812f4.png)

**Reply by roberthstrand** [reply]
Well, I'm a man of excellent taste and I have to admit that the logo is great :D The arrow seals the deal

**Reply by todaywasawesome** [reply]
Great work!

**Comment by scottrigby** [comment]
🎉 Cross-posted from [CNCF SIG-App Delivery mailing list](https://lists.cncf.io/g/cncf-sig-app-delivery):

>📣 **We have reached consensus** on logo designs for both the GitOps Working Group and OpenGitOps!
>
>👀 The current designs are in [this discussion thread](https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-544509) (be sure to click "view more" to see everything).
>
>🔎 If you're interested in the detailed steps of rolling out the designs, follow [this issue](https://github.com/gitops-working-group/gitops-working-group/issues/100).
>
>👏 Thanks again @iboonox and everyone who participated!

**Reply by christianh814** [reply]
Now we need a 128 pixels by 128 pixels version @iboonox for all the slack channels :D

**Comment by cdavisafc** [comment]
I've missed Scott's deadline, but will still chime in that I absolutely LOVE these! I'm not fond of the one with the photo of a person in it but I'm seeing the final two as the official one.

GREAT work everyone!!!

**Reply by scottrigby** [reply]
>I'm seeing the final two as the official one

Yes! With a slight adjustment of the OGO arrow here 🙂 https://github.com/gitops-working-group/gitops-working-group/discussions/63#discussioncomment-549888
